presto-docs/src/main/sphinx/release/release-0.254.rst

```diff
@@ -0,0 +1,34 @@
+=============
+Release 0.254
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a bug where queries that have both remote functions and a local function with only constant arguments could throw an IndexOutOfBoundException during planning. The bug was introduced in release 0.253 by :pr:`16039`.
+* Add fragment result caching support for ``UNNEST`` queries.
+* Add :func:`poisson_cdf` and :func:`inverse_poisson_cdf` functions.
+* Add memory tracking in ``TableFinishOperator`` which can be enabled by setting the ``table-finish-operator-memory-tracking-enabled`` configuration property to ``true``. Enabling this property can help investigating GC issues on the coordinator, by allowing us to debug whether stats collection uses a lot of memory.
+* Remove spilling strategy ``PER_QUERY_MEMORY_LIMIT`` and instead add configuration property ``experimental.query-limit-spill-enabled`` and session property ``query_limit_spill_enabled``. When ``query_limit_spill_enabled`` is set to ``true`` and the spill strategy is not ``PER_TASK_MEMORY_THRESHOLD``, then we will spill whenever a query uses more than the per-node total memory limit in combined revocable and non-revocable memory.  We will also still spill whenever the memory pool exceeds the spill threshold. This fixes an issue where using the ``PER_QUERY_MEMORY_LIMIT`` spilling strategy could prevent the oom killer from running when the memory pool was full. The issue is still present for the ``PER_TASK_MEMORY_THRESHOLD`` spilling strategy.
+
+Hive Changes
+____________
+* Fix a bug where the files would not be sorted when inserting into bucketed sorted tables with Glue.
+* Add support for partition cache validation. This can be enabled by setting the ``hive.partition-cache-validation-percentage`` configuration property. Partition cache validation allows us to validate the value returned from partition cache with the actual value from Metastore. The default value is 0.0 which means there is no validation.
+* Add support for allowing to match columns between table and partition schemas by names when the configuration property ``hive.parquet.use-column-names`` or the hive catalog session property ``parquet_use_column_names`` is set to true. By default they are mapped by index.
+* Add support for configuring the Glue endpoint URL :doc:`/connector/hive`.
+* Add support for accessing tables in Glue metastore that do not have a table type.
+* Add support for the S3 Intelligent-Tiering storage class for writing data. The S3 storage class can be configured using the configuration property ``hive.s3.storage-class``. Supported values are ``STANDARD`` and ``INTELLIGENT_TIERING``, and the default value is ``STANDARD``.  :doc:`/connector/hive`.
```

Comment:
I don't think we need to mention the default value since that would already be there.

```
Add support for the S3 Intelligent-Tiering storage class writing data. This can be enabled by setting the configuration property ``hive.s3.storage-class`` to ``INTELLIGENT_TIERING``.
```